docs/articles/new-expensify/get-paid-back/Distance-Requests.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Distance Requests
+description: How to create a distance request and request reimbursement for mileage
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# Overview
+
+Expensify allows you to request reimbursement for mileage by creating a distance request from a map. You can send a distance request in Expensify's mobile, desktop, or web app.
+
+
+# How to create and send a distance request
+
+1. Click the green '+' button and select Request Money.
```

Comment:
Why put the `+` in backticks but not `Request Money` either?

Reply:
Yeah I don't think we need the quotes

Reply:
I'll remove the single quotes.